src/vic.h

```diff
@@ -77,6 +77,9 @@
 extern char vic_buffer[VIC_BUFFER_SIZE + 1];
 extern uint8_t vic_buffer_len;
 
+#define vic_printf(format, ...) do { char _vic_buff[VIC_BUFFER_SIZE];\
```

Comment:
@vargac I am wondering how well will this work in the real world: `VIC_BUFFER_SIZE` is pretty big and it might be that you will not often have enough space to conduct this whole operation.

Am I missing something obvious here? 

cc @burlito